common/src/main/java/com/LubieKakao1212/opencu/common/device/TrackerBase.java

```diff
@@ -94,15 +129,14 @@ public void tick(BlockEntityModularFrame frame, IDeviceState state, World world,
                     rsout.result = rsct == RedstoneControlType.PULSE ? rsPulser.shouldActivate() : true;
                     pulse.result = true;
 
-                    //var delta = new Vector3m(pos.toCenterPos().subtract(nearestEntity.getBoundingBox().getCenter()));
-                    var target = new Vector3m(nearestEntity.getBoundingBox().getCenter());
+                    var target = new Vector3m(nearestEntity.getBoundingBox().getCenter().add(0, nearestEntity.getStandingEyeHeight()/2, 0));
```

Comment:
This will point to the top of the entity since `getCenter()` already returns the center of the bounding box  

```suggestion
                    var target = new Vector3m(nearestEntity.getBoundingBox().getCenter());
```

Reply:
This is intended, at a slightly larger distance (especially diagnal) the arrow will basically always miss because its aiming at the center and the arrows droop, aiming at the entity top fixes this.

Reply:
How does it behave at close range and how does it behave with projectiles that don't have gravity like fireworks?

Reply:
Close range with arrows is as you would expect, with fireworks it should also be fine with it aiming at the top of the head as minecraft naturally aims down slightly, however we can also test this more and see if a slightly larger denominator would make a happy medium